endpoints/cookie_sync_test.go

```diff
@@ -787,122 +790,221 @@ func TestCookieSyncParseRequest(t *testing.T) {
 }
 
 func TestSetLimit(t *testing.T) {
-	coopSync := true
-
 	testCases := []struct {
 		description     string
 		givenRequest    cookieSyncRequest
-		givenConfig     config.UserSync
-		expectedError   string
+		givenAccount    *config.Account
 		expectedRequest cookieSyncRequest
 	}{
 		{
-			description: "Test that Max Limit and Default Coop Sync are Applied",
+			description: "Default Limit is Applied (request limit = 0)",
 			givenRequest: cookieSyncRequest{
-				Account: "TestAccount",
-				Limit:   42,
+				Limit: 0,
 			},
-			givenConfig: config.UserSync{
-				Cooperative: config.UserSyncCooperative{
-					PriorityGroups: [][]string{{"a", "b", "c"}},
+			givenAccount: &config.Account{
+				CookieSync: config.CookieSync{
+					DefaultLimit: &[]int{20}[0],
```

Comment:
IMHO: This is a really confusing way to convert an int to a pointer. The array is disorienting.

You can create a variable within the function scope and reference it here, like `DefaultLimit: &int20` or write a simple test helper method similar to what's in the mxmcherry library.